src/main/proto/contract.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package proto;
+
+option java_multiple_files = true;
+option java_package = "generated";
+option java_outer_classname = "ProductsProtos";
+
+service product{
+  rpc addProduct(addProductRequest) returns (addProductResponse) {};
+  rpc listProduct(listProductRequest) returns (stream listProductResponse) {};
+  rpc listById(listByIdRequest) returns (listByIdResponse) {};
+}
+message listProductRequest{
+}
+message listProductResponse{
+  int32    id = 1;
```

Comment:
Too much empty spaces between variable type and its name.

Please check this all over this file

Reply:
Done.